crates/stages/src/test_utils/test_db.rs

```diff
@@ -252,3 +252,15 @@ impl TestTransaction {
         })
     }
 }
+
+struct InsertOpts {
+    headers: Vec<SealedHeader>,
+    blocks: Vec<SealedBlock>,
+    difficulty: bool,
+}
+
+impl From<Vec<SealedHeader>> for InsertOpts {
+    fn from(headers: Vec<SealedHeader>) -> Self {
+        InsertOpts { headers, blocks: Vec::new(), difficulty: false }
+    }
+}
```

Comment:
need to remove this

was early thought on how we could start having a unified config for instantiating test chains